charts/datahub/subcharts/datahub-frontend/templates/_helpers.tpl

```diff
@@ -66,7 +66,7 @@ Create the name of the service account to use
 Return the appropriate apiVersion for ingress.
 */}}
 {{- define "datahub-frontend.ingress.apiVersion" -}}
-{{- if semverCompare ">=1.22-0" .Capabilities.KubeVersion.Version -}}
+{{- if semverCompare ">=1.19-0" .Capabilities.KubeVersion.Version -}}
```

Comment:
This is to take advantage of `networking.k8s.io/v1` when it first becomes available rather than have the cluster upgraded underneath us and end up with an unsupported API version. This API becomes stable in v1.19 (https://kubernetes.io/docs/concepts/services-networking/ingress/)